### DISCUSSION TITLE
How to pass custom request header to API Gateway integration using SAM Template?

### DISCUSSION THREAD

**Discussion by idxakshay** [aws/serverless-application-model#2704] [Questions]
I want to pass custom headers to Lambda Integration from Sam Template. Is there any way to do it from the SAM template?

I know this is possible from the parameter mapping from the API Gateway console, but I could not find a way to achieve the same using SAM.

![image](https://user-images.githubusercontent.com/71782108/191766920-9cfe247d-58df-4cf4-87b7-4b7bb0ac9b50.png)

**Comment by sriram-mv** [comment]
Is this what you are looking for: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-property-function-requestparameter.html?

**Comment by pethaniakshay** [comment]
> Is this what you are looking for: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-property-function-requestparameter.html?

Yes, Thats exactly i want but how to define the value?

I want to pass the value also specificed in sam template from api gateway

**Comment by aaythapa** [comment]
We won't be prioritizing this for now
